apps/web/app/api/social/og/image/route.tsx

```diff
@@ -89,7 +93,8 @@ async function handler(req: NextRequest) {
             status: 200,
             headers: {
               "Content-Type": "image/png",
-              "Cache-Control": "public, max-age=3600, stale-while-revalidate=86400",
+              "Cache-Control": `public, max-age=31536000, immutable, s-maxage=31536000, stale-while-revalidate=31536000`,
```

Comment:
"immutable" says: "Don't revalidate this resource until the ETag changes"
Browser cache: 1 year
CDN cache: 1 year
ETag: `"app-${APP_OG_IMAGE_VERSION}"`, `"meeting-${MEETING_OG_IMAGE_VERSION}", `"generic-${GENERIC_OG_IMAGE_VERSION}"`